dev/password-field.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+
+    <script type="module">
+      import '@vaadin/password-field';
+    </script>
+  </head>
+
+  <body>
+    <vaadin-password-field label="Required" required></vaadin-password-field>
+
+    <div>Is valid? <span id="validity"></span></div>
+
+    <script>
+      const field = document.querySelector('vaadin-password-field');
+      const validity = document.querySelector('#validity');
+
+      field.addEventListener('change', (e) => {
```

Comment:
This component has the same issue as the `vaadin-email-field` component. Maybe the `change` event is fired by the component before its validation status is updated.
```suggestion
      field.addEventListener('focusout', (e) => {
```

Reply:
Removed this listener.